django_stomp/execution.py

```diff
@@ -16,13 +17,13 @@
 wait_to_connect = int(getattr(settings, "STOMP_WAIT_TO_CONNECT", 10))
 durable_topic_subscription = eval_str_as_boolean(getattr(settings, "STOMP_DURABLE_TOPIC_SUBSCRIPTION", False))
 listener_client_id = getattr(settings, "STOMP_LISTENER_CLIENT_ID", None)
-if not durable_topic_subscription:
-    if listener_client_id:
-        listener_client_id = f"{listener_client_id}-{uuid.uuid4().hex}"
+publisher_name = "django-stomp-another-target"
+if not durable_topic_subscription and listener_client_id:
+    listener_client_id = f"{listener_client_id}-{uuid.uuid4().hex}"
 
 
-def start_processing(destination_name: str, callback_str: str, is_testing=False, testing_disconnect=True):
-
+def start_processing(destination_name: str, callback_str: str, is_testing=False, testing_disconnect=True, **kwargs):
+    param_to_callback = kwargs.get("param_to_callback")
```

Comment:
If we are accepting the generic param _kwargs_, shouldn't we pass it **as is** to the **callback_function**? Otherwise wouldn't it be better to have a **single parameter** named **param_to_callback** on the function _start_processing_?

Reply:
Good point, IGU!